.lintstagedrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "*": "prettier --ignore-unknown --write"
```

Comment:
Does this call `pnpm prettier` under the hood, or only work if `prettier` is globally installed?